LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
OK, so I guess this is the downside of using GPL 3 software is that it forces us to change our license (see https://www.apache.org/licenses/GPL-compatibility.html). It's fine, we'll move to GPL 3 for just this in order to use that theme.

Reply:
Yep, we are forced to use the GPL here if that's not a problem.

Reply:
I think it's fine for this, we'd need a bigger discussion if this was for one of our apps or libraries.

Reply:
Yep, makes sense.